src/statsmodels/formula.jl

```diff
@@ -37,6 +37,12 @@ type ModelFrame
     df::AbstractDataFrame
     terms::Terms
     msng::BitArray
+    ## mapping from df keys to contrasts.  Rather than specify types allowed,
```

Comment:
The first part of the comment is good, but no need to give the rationale for the design here.

Maybe make this a `Dict{Symbol, AbstractContrasts}`?